### PR TITLE
Add information about ONNX backend to the docs

### PR DESCRIPTION
**Pull request by andrey-churkin**
### Details:
 - The quantize with accuracy control supports the ONNX backend. I have added information on how to use quantize with accuracy control for `onnx.ModelProto`.

### Tickets:
 - *133388*

### CODE REVIEW

**Comment by andrey-churkin** [comment]
@MaximProshin @tsavina Please review

**Comment by MaximProshin** [comment]
@andrey-churkin @tsavina , can you share the html version of that?

**Comment by tsavina** [comment]
> @andrey-churkin @tsavina , can you share the html version of that?

http://openvino-doc.iotg.sclab.intel.com/onnx-doc/quantization_w_accuracy_control.html

File: docs/articles_en/openvino-workflow/model-optimization-guide/quantizing-models-post-training/quantizing-with-accuracy-control.rst [hunk 1]
```diff
@@ -38,10 +38,18 @@ This step is similar to the :doc:`Basic 8-bit quantization <basic_quantization_f
...
+      .. doxygensnippet:: docs/optimization_guide/nncf/ptq/code/ptq_aa_onnx.py
+         :language: python
+         :fragment: [dataset]
+
 Prepare validation function
 ############################
 
-Validation function receives ``openvino.CompiledModel`` object and validation dataset and returns accuracy metric value. The following code snippet shows an example of validation function for OpenVINO model:
+The validation function takes two arguments: a model object and a validation dataset, and it returns the accuracy metric value. The type of the model object is different for different frameworks. In OpenVINO, it is an ``openvino.CompiledModel``. In ONNX, it is an ``onnx.ModelProto``.
+The following code snippet shows an example of validation function for OpenVINO and ONNX framework:
```
[the remark this answers is not in the document]
Comment (andrey-churkin) [reply docs/articles_en/openvino-workflow/model-optimization-guide/quantizing-models-post-training/quantizing-with-accuracy-control.rst]: Fixed

File: docs/articles_en/openvino-workflow/model-optimization-guide/quantizing-models-post-training/quantizing-with-accuracy-control.rst [hunk 2]
```diff
@@ -38,10 +38,18 @@ This step is similar to the :doc:`Basic 8-bit quantization <basic_quantization_f
...
+
+      .. doxygensnippet:: docs/optimization_guide/nncf/ptq/code/ptq_aa_onnx.py
+         :language: python
+         :fragment: [dataset]
+
 Prepare validation function
 ############################
 
-Validation function receives ``openvino.CompiledModel`` object and validation dataset and returns accuracy metric value. The following code snippet shows an example of validation function for OpenVINO model:
+The validation function takes two arguments: a model object and a validation dataset, and it returns the accuracy metric value. The type of the model object is different for different frameworks. In OpenVINO, it is an ``openvino.CompiledModel``. In ONNX, it is an ``onnx.ModelProto``.
```
[the remark this answers is not in the document]
Comment (andrey-churkin) [reply docs/articles_en/openvino-workflow/model-optimization-guide/quantizing-models-post-training/quantizing-with-accuracy-control.rst]: Fixed

**Review by tsavina — APPROVED** [review]

**Pull request merged**